k9mail/src/main/java/com/fsck/k9/fragment/MessageListAdapter.java

```diff
@@ -80,6 +84,7 @@ public View newView(Context context, Cursor cursor, ViewGroup parent) {
         holder.date = (TextView) view.findViewById(R.id.date);
         holder.chip = view.findViewById(R.id.chip);
 
+        mContext = context;
```

Comment:
Storing the context is generally not a good idea and I don't think it's needed here.